### PR TITLE
Master res currency rate set default 1 axj

### PR DESCRIPTION
**Pull request by axj-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

Task:
https://www.odoo.com/web#id=37899&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

Pad:
https://pad.odoo.com/p/openerp-project.task-EWUDP6DE3T

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by rgo-odoo** [comment]
LGTM

File: odoo/addons/base/tests/test_float.py [hunk 1]
```diff
@@ -160,9 +160,9 @@ def try_roundtrip(value, expected, date):
 
         # res.currency.rate uses 6 digits of precision by default
         try_roundtrip(2.6748955, 2.674896, '2000-01-01')
-        try_roundtrip(-2.6748955, -2.674896, '2000-01-02')
+        try_roundtrip(0.6748955, 0.674896, '2000-01-02')
```
Comment (qdp-odoo) [inline odoo/addons/base/tests/test_float.py]: @rco-odoo we want to remove the possibility to have negative rates in currencies (meaningless)... So you might want to use another object in that test? or can we just remove the lines testing negative values?

Comment (qdp-odoo) [reply odoo/addons/base/tests/test_float.py]: as discussed: we'll comment those lines and put a TODO statement for that them being re-implemented with proper dedicated model.

**Comment by qdp-odoo** [comment]
landed in master, for saas19, at revision https://github.com/odoo/odoo/commit/298491597cf30418d5e0355eef1fdb12289171e4

**Pull request closed without merging**